docs/cmd/prompt.md

```diff
@@ -1,57 +1,66 @@
 
-* Auto-completion for all commands names and their flags.
-* Auto-completion for flags with xpath values ( `--path`, `--prefix`, `--model`,...). The xpaths are generated from models suplied using `prompt` command `--file` and `--dir` (optional) flags.
-* Auto-completion for flags with enum values `--format`, `--encoding`, ...
-* Auto-completion for flags taking file or directory values.
+* All `gnmic` [commands names and their flags are suggested](../advanced/prompt_suggestions.md#commands-and-flags-suggestions).
+* Values for the flags that rely on YANG-defined data (like `--path`, `--prefix`, `--model`,...) will be dynamically suggested, we call this feature [YANG-completions](../advanced/prompt_suggestions.md#yang-completions).  
+The auto-completions are generated from the YANG modules d with the `--file` and `--dir` flags.
+* Flags with the fixed set of values (`--format`, `--encoding`, ...) will get their [values suggested](../advanced/prompt_suggestions.md#enumeration-suggestions).
+* Flags that require a [file path value will auto-suggest](../advanced/prompt_suggestions.md#file-path-completions) the available files as the user types.
 
 
 ### Usage
 
 `gnmic [global-flags] prompt [local-flags]`
 
-
-### Flags 
+### Flags
 
 #### file
-
-File or directories pointing to yang modules from which to generate gnmi paths.
+A path to a YANG file or a directory with YANG files which `gnmic` will use to generate auto-completion for YANG-defined data (paths, models).
 
 Multiple `--file` flags can be supplied.
 
 #### dir
+A path to a directory which `gnmic` would recursively traverse in search for the additional YANG files which may be required by YANG files specified with `--file` to build the YANG tree.
 
-Direcotries where to search for yang modules included and imported in modules under `--file`
+Can also point to a single YANG file instead of a directory.
 
 Multiple `--dir` flags can be supplied.
 
-#### exclude
+#### description-with-prefix
+When set, the description field of the suggestion box will have a module prefix name before the element description.
 
-The `--exclude` flag is used to specify the yang modules to be excluded from path generation
+#### description-with-types
+When set, the description field of the suggestion box will have a YANG type information provided for the elements.
+
+#### exclude
+The `--exclude` flag specifies the YANG module __names__ to be excluded from the path generation when YANG modules names are clashed.
 
 Multiple `--exclude` flags can be supplied.
 
 #### max-suggestions
+The `--max-suggestions` flag sets the number of lines that the suggestion box will display without scrolling.
 
-The `--max-suggestions` flag is used to limit the size the the terminal box showing command completion suggestions. (uint16)
+Defaults to 10.
 
-#### description-bg-color
+#### suggest-all-flags
+The `--suggest-all-flags` makes `gnmic` prompt to suggest both global and local flags for a sub-command.
```

Comment:
`to suggest` ==> `suggest`

Reply:
done in a0d15077bf6f8e91f99d53aa7f65599e3a72e450